commands/rungearoutake.py

```diff
@@ -0,0 +1,20 @@
+import wpilib
+from wpilib.command import Command
+import subsystems
+import robotmap
+
+class RunGearOutake(Command):
+
+    def __init__(self, position):
+        super().__init__('Gear Outake')
+        self.requires(subsystems.gearoutake)
+        self.position = 0.0
```

Comment:
You're not actually using the `position` argument to set the instance variable `position`. Change this line to:
`self.position = position`.